### PR TITLE
Release/v0.4.6: Documentation links fixed

### PR DESCRIPTION
**Pull request by matbmoser**
# What is new?

## Updated
- Updated documentation, corrected links.

### CODE REVIEW

File: docs/admin guide/Admin_Guide.md [hunk 1]
```diff
@@ -2,10 +2,8 @@
 
 ![C-X Logo](./CXLogo.png)  
 
-Version: v1.1 </br>
-Latest Revision Jan 18, 2023
-
->Some links might not be accesible as they lead to a private confluence. If you need access please reach out to the dev Team and request what information you need for what reason.
+Version: v1.2 </br>
```
Comment (matbmoser) [inline docs/admin guide/Admin_Guide.md:5]: Updated version of document since links changed

File: docs/arc42/Arc42.md [hunk 2]
```diff
@@ -18,10 +18,9 @@
 
 ![C-X Logo](./CXlogo.png) ![acr24 logo](./arc24logo.png)  
 
-Version: v2.0
-Latest Revision Dec. 13, 2022
+Version: v2.1 <br>
```
Comment (matbmoser) [inline docs/arc42/Arc42.md:21]: Updated version of document since links changed

File: DEPENDENCIES_BACKEND [hunk 3]
```diff
@@ -128,4 +128,4 @@ maven/mavencentral/org.springframework/spring-webflux/6.0.4, Apache-2.0, approve
 maven/mavencentral/org.springframework/spring-webmvc/6.0.4, Apache-2.0, approved, #5944
 maven/mavencentral/org.webjars/swagger-ui/4.15.5, Apache-2.0 AND MIT, approved, #5921
 maven/mavencentral/org.webjars/webjars-locator-core/0.52, MIT, approved, clearlydefined
-maven/mavencentral/org.yaml/snakeyaml/2.0, , restricted, clearlydefined
+maven/mavencentral/org.yaml/snakeyaml/2.0, Apache-2.0 AND (Apache-2.0 AND BSD-3-Clause AND EPL-1.0 AND GPL-2.0-or-later AND LGPL-2.1-or-later), restricted, #7275
```
Comment (matbmoser) [inline DEPENDENCIES_BACKEND:131]: Added the ticket with the revision

**Review by thorstendikmann — APPROVED** [review]
- Only documentation files have been changed
  - .md-Files, including readding user man
  - JPGs including outdated links
  
- Yaml-files: internal version increased

**Pull request merged**